### ISSUE TITLE
Suppress scientific notation of numeric values in GeoData (and potentially in all file exports, if prefered)

### ISSUE THREAD

**Issue by dgustafsson** [rcapell/HYPEtools#127]
I struggled a  bit to export a modified GeoData with right number of decimals and avoiding scientific notation (for some reason I wanted the export not to have scientific notation), but then I found this information that it can be controlled by argument scipen to fwrite:

https://stackoverflow.com/questions/43227387/how-can-i-disable-scientific-notation-in-fwrite-data-table-in-r

So I simply added this argument to WriteGeoData and passed it on to the call to fwrite.

A penalty of 999 seems to be recommended as a start.

**Comment by c-brendel** [comment]
Hej David,

Using options(scipen = 999) disables scientific notation for everything in R and not just for fwrite. I suppose I could add something that sets options(scipen = 999) before writing and then sets it back to the original value after writing. What do you think? I don't want to just set the scipen option and then have it mess up the rest of people's scripts if they want scientific notation later.

Conrad

**Comment by c-brendel** [comment]
Ah, nevermind. I see that it is an option within the fwrite function now!

**Comment by rcapell** [comment]
Making arguments of wrapped functions available is always possible, I'd say. Added 'scipen' to WriteGeoData now.
Sidenote: I first tried to be clever and use the dots argument ('...') to pass unknown arguments to fread. But it turned out to be non-trivial to write bug-safe code (see [this tutorial](https://www.natedayta.com/2018/12/31/using-r-s-elipse-dots-to-write-better-wrappers/), particularly the Edit-comment at the end)
